log-config.yaml

```diff
@@ -1,7 +1,11 @@
-severity: Debug
+rotation:
+    logLimit: 2097152 # 2MB
+    keepFiles: 20
+severity: Error
```

Comment:
I am not sure we will catch useful information with severity Error for debugging. 

For deployment this makes sense though.